docs/sources/articles/b2d_volume_resize.md

```diff
@@ -6,21 +6,39 @@ page_keywords: boot2docker, volume, virtualbox
 
 If you're using Boot2Docker with a large number of images, or the images you're
 working with are very large, your pulls might start failing with "no space left 
-on device" errors when the Boot2Docker volume fills up. The solution is to 
-increase the volume size by first cloning it, then resizing it using a disk 
-partitioning tool. 
+on device" errors when the Boot2Docker volume fills up. There are two solutions
+in this case:
+
+## Solution 1: Add the `DiskImage` property in boot2docker profile
+
+The boot2docker binary reads configuration from `$BOOT2DOCKER_PROFILE` if set, or `$BOOT2DOCKER_DIR/profile` or `$HOME/.boot2docker/profile` or (on Windows) `%USERPROFILE%/.boot2docker/profile`. boot2docker config will tell you where it is looking for the file, and will also output the settings that are in use, so you can initialise a default file to customize using `boot2docker config > ~/.boot2docker/profile`.
+Add the following lines to `$HOME/.boot2docker/profile`:
+
+    # Disk image size in MB
+    DiskSize = 50000
+
+And run:
+
+    boot2docker poweroff
+    boot2docker destroy
+    boot2docker init
+    boot2docker up
+
+## Solution 2: Increase the volume size
+
+The solution increase the volume size by first cloning it, then resizing it using a disk partitioning tool. 
```

Comment:
There's still a word or more missing here. Should it read "The solution is to increase the volume size..."? Don't want to hold up this PR, however, so we can always fix it in a follow-on.